db/migrate/20230921082209_change_subject_limited_to_not_null_in_countries.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeSubjectLimitedToNotNullInCountries < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :countries, :subject_limited, false
```

Comment:
Rather than introducing a new migration, we can alter the existing one as it's not merged in to `main` yet and hasn't been run in any of our environments, that keeps the migrations clean and means there will never be a time where the column is `nullable`. It also means rolling back can happen in one step rather than two.